.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: go
+
+go:
+  - 1.14.x
+
+cache:
+  directories:
+    - "$HOME/.cache/go-build"
+    - "$GOPATH/pkg/mod"
+
+matrix:
+  include:
+    - name: Verify deps & Lint
+      script:
+        - rm -rf $HOME/.cache/golangci-lint || true
+        - make verify build
+        - make lint
+    - name: Unit Tests
+      script:
+        - make test-import
```

Comment:
this is failing bc of timeout:
`panic: test timed out after 5m0s`

Reply:
yes, shall we increase timeout then ?

Reply:
I increased it to 10m, lets see if we get a green run now on the first try! :+1: